cmd/root.go

```diff
@@ -57,6 +57,8 @@ func AddFlags(flagSet *pflag.FlagSet, cfg *config.Config) (configPath *string) {
 		cfg.LayerAvgSize, "Layer Avg size")
 	flagSet.BoolVar(&cfg.PprofHTTPServer, "pprof-server",
 		cfg.PprofHTTPServer, "enable http pprof server")
+	flagSet.StringVar(&cfg.PprofHTTPServerListener, "pprof-listener", cfg.PprofHTTPServerListener,
+		"Socket for pprof server, not safe to expose publicly")
```

Comment:
This is not exactly "socket", rather "listen address"

Reply:
Changed anyway.